application/domain/domain.go

```
@@ -119,6 +138,18 @@ var Env struct {
 	SamlCheckResponseSigning        bool   `default:"true" split_words:"true"`
 	SamlSignRequest                 bool   `default:"true" split_words:"true"`
 	SamlRequireEncryptedAssertion   bool   `default:"true" split_words:"true"`
+
+	// Ported from WeCarry's domain.go so that storage.go works
+	AwsRegion          string `default:"us-east-1" split_words:"true"`
```

Do we really want local dev/test values as defaults on these? I'd rather see no default and the app fail to start/run if the env vars are not present. That'll save us from weird problems in deployed environments down the road.

---

:+1: